Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py

```diff
@@ -114,11 +114,10 @@ def get_incidents_by_keys(similar_incident_keys, time_field, incident_time, inci
 
     similar_keys_list = []
     for key, value in similar_incident_keys.items():
+        value = value.encode('utf-8')
         compare_str = '{}:={}' if str(value).isdigit() else '{}="{}"'
-        similar_keys_list.append(compare_str.format(key, str(value)
-                                 .replace('"', r'\"')
-                                 .replace("\n", "\\n").
-                                 replace("\r", "\\r")))
+        similar_key = compare_str.format(key, str(value).replace('"', r'\"').replace("\n", "\\n").replace("\r", "\\r"))
```

Comment:
This isn't very readable at the moment in my opinion.
However, leaving this to your judgement @teizenman @ChanochShayner